<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eDRC::Internal::CrListViewer</class>
 <widget class="QWidget" name="eDRC::Internal::CrListViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>350</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="Views::TreeView" name="treeView">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Views::TreeView</class>
   <extends>QTreeView</extends>
   <header location="global">listviewplugin/treeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
